pkg/lib/config/saml.go

```diff
@@ -134,6 +138,15 @@ var _ = Schema.Add("SAMLSigningSignatureMethod", `
 
 type SAMLSigningSignatureMethod string
 
+func (m SAMLSigningSignatureMethod) ToDsigSignatureMethod() string {
+	switch m {
+	case SAMLSigningSignatureMethodRSASHA256:
+		return dsig.RSASHA256SignatureMethod
```

Comment:
This library already uses the correct constant. https://github.com/russellhaering/goxmldsig/blob/main/xml_constants.go#L46C2-L46C26

And I discovered they call it `signature method` instead of `signature algorithm`. So in our config, we should make it like

```
signature_method: "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"
```

---


Rename it back to `signature_method`, and correct the enum values.

Reply:
ok. Just to note:
There are two thing in xmldsig, the signing method and digest method. Our config now controls both.